### PR TITLE
Add post-dev15.5-contrib to MyGet publish list

### PR DESCRIPTION
**Pull request by agocke**
Looks like I didn't configure the branch to actually upload NuGet packages.

### CODE REVIEW

File: build/config/PublishData.json [hunk 1]
```diff
@@ -7,6 +7,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.5", "dev15.5p3" ]
         },
+        "post-dev15.5-contrib": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.6.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v3/index.json" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev15.5" ]
```
Comment (jasonmalinowski) [inline build/config/PublishData.json]: This is definitely wrong.

File: build/config/PublishData.json [hunk 2]
```diff
@@ -7,6 +7,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.5", "dev15.5p3" ]
         },
+        "post-dev15.5-contrib": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.6.*",
```
Comment (jcouv) [inline build/config/PublishData.json]: Should this be 2.7.*?

**Comment by jasonmalinowski** [comment]
@agocke FYI that we're also killing that branch in the next day or two...

File: build/config/PublishData.json (hunk 1 above)
Comment (jcouv) [inline build/config/PublishData.json]: dev15.6?

**Comment by agocke** [comment]
@jasonmalinowski I literally just need a Microsoft.NETCore.Compilers package to do a toolset update. Is there something else I should do?

**Comment by agocke** [comment]
@jasonmalinowski Update channel to dev15.6 but kept the version the same. Does this sound right?

**Pull request closed without merging**